scripts/linkbot/monitor-ub-server.js

```diff
@@ -26,12 +26,14 @@ JSDOM.fromURL(url, {}).then(dom => {
     var refs = document.getElementsByTagName("a");
     var matches = Array.from(refs)
         .filter(ref => ref.textContent.match(reg))
+        .filter(ref => parseInt(ref.textContent.match(reg).groups.release) % 2 == 0) // Only LTS
+        .filter(ref => !ref.textContent.match(reg).groups.minor) // Ignore the minor number
```

Comment:
Is this enough? You also need to check for "04", no? They have a "10" release which is not LTS.

Reply:
It's done here:
https://github.com/mendersoftware/mender-convert/blob/f075b8f137c39697389468c28dfcf5dcdc210cb5/scripts/linkbot/monitor-ub-server.js#L8

Reply:
Aha, thanks! Strictly speaking you could then match this in the same regex with `(?<release>[0-9][02468])`, but I leave it to you to decide whether that's tidier or not.

Reply:
Good point! Giving it a spin :)

Reply:
Working fine ;)